go.mod

```diff
@@ -3,24 +3,25 @@ module sigs.k8s.io/cluster-api-provider-aws
 go 1.13
 
 require (
-	github.com/aws/aws-sdk-go v1.31.3
+	github.com/aws/aws-sdk-go v1.31.6
 	github.com/awslabs/goformation/v4 v4.8.0
 	github.com/davecgh/go-spew v1.1.1
 	github.com/go-logr/logr v0.1.0
-	github.com/golang/mock v1.3.1
-	github.com/onsi/ginkgo v1.12.0
-	github.com/onsi/gomega v1.9.0
+	github.com/golang/mock v1.4.3
+	github.com/onsi/ginkgo v1.12.2
+	github.com/onsi/gomega v1.10.1
 	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v0.0.6
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073
-	golang.org/x/net v0.0.0-20200301022130-244492dfa37a
-	gopkg.in/yaml.v2 v2.2.8
+	golang.org/x/crypto v0.0.0-20200510223506-06a226fb4e37
+	golang.org/x/net v0.0.0-20200520182314-0ba52f642ac2
+	gopkg.in/yaml.v2 v2.3.0
```

Comment:
since we're adding sigs.k8s.io/yaml, can we replace the existing use of gopkg.in/yaml.v2 with sigs.k8s.io/yaml as well to avoid pulling in two separate yaml libraries?